### ISSUE TITLE
Ensure `spec.features.collection_type_spec` passes on `.koppie`

### ISSUE THREAD

**Issue by no-reply** [samvera/hyrax#6190]
`spec.features.collection_type_spec` is failing on koppie builds. there may be a number of reasons for this:

  1. the test setup uses ActiveFedora, and should be refactored to use Valkyrie;
  2. the tests themselves test behavior that only applies to ActiveFedora, and should be turned off by adding the `:active_fedora` rspec metadata;
  3. the unit under test depends on ActiveFedora and needs to be rewritten to use Valkyrie throughout;
  4. some combination of the above.

----

```sh
docker-compose -f docker-compose-koppie.yml build
docker-compose -f docker-compose-koppie.yml up
docker-compose -f docker-compose-koppie.yml exec -w /app/samvera/hyrax-engine app sh -c "bundle exec rspec spec/features/collection_type_spec.rb"
```

**Comment by sdellis** [comment]
The admin_user has the following groups: ["registered"] whereas it should have ["admin"] so the feature tests are not authorized to access the page they are testing. It seems like much of the failing tests here may have to do with the fact that the Factory is not creating the right group/credentials to allow access.

**Comment by sdellis** [comment]
Interestingly, [Dassie's user.rb file](https://github.com/samvera/hyrax/blob/main/.dassie/app/models/user.rb) does not have the [Role Management code that Koppie's user.rb file has](https://github.com/samvera/hyrax/blob/main/.koppie/app/models/user.rb#L4C1-L5C43), yet it's the Koppie tests that are failing because of authorization issues.

This is in Koppie's, but not in Dassie's user model:
```
 # Connects this user object to Role-management behaviors.
  include Hydra::RoleManagement::UserRoles
```
Adding this code to Dassie does not resolve the error and removing it from Koppie results in:
`undefined method 'admin?' for #<User>`

**Comment by no-reply** [comment]
@sdellis i noticed `spec.models.user_spec` also fails on `.koppie` with a related error, which might just confirm what you already know, or might be a more helpful starting place?

**Comment by sdellis** [comment]
Closing with one final failing test. 

Ref #5522

**Comment by no-reply** [comment]
@sdellis i think this is still open, pending a fix for/merge of #6268?

**Comment by hackartisan** [comment]
resolved by #6268
